honeybee_3dm/model.py

```diff
@@ -0,0 +1,67 @@
+"""Creating Honeybee model objects from rhino3dm surfaces and closed volumes"""
+import os
+
+# The Rhino3dm library provides the ability to access content of a Rhino3dm
+# file from outside of Rhino
+import rhino3dm
+
+# Importing core Honeybee dependency
+from honeybee.model import Model
+from honeybee.typing import clean_string
+
+# Importing dependencies from Honeybee-3dm package
+from .room import import_rooms
+from .face import import_faces
+from .grid import import_grids
```

Comment:
Yes. We have a lot more to do here. Mostapha wanted me to create a PR so that you both can review it. And I can see that this process is very important.